### PR TITLE
Add map to DAShboard

### PR DESCRIPTION
**Pull request by hallgchris**
image

Test with `mqtt_test.py` as usual, shouldn't need any setup.

### CODE REVIEW

**Comment by hallgchris** [comment]
One thing I want people's opinion on - right now for narrower browser windows, it looks like this:

image

I made the second row split earlier in the first row, as otherwise the maps would be tiny (like those first two graphs). However, it means the cadence chart is excessively big. Thoughts?

File: client/js/index.js [hunk 1]
```diff
@@ -166,3 +194,52 @@ function updateTextMode() {
   }
 }
 updateTextMode(); // State of switch is kept after refresh, so check on page load
+
+function setupMap() {
+  // .nav-link.active from common.css
+  const highlightColor = '#007bff';
```
Comment (khanguslee) [inline client/js/index.js:200]: We should create a constants file later on that stores stuff like this.

**Comment by khanguslee** [comment]
> I made the second row split earlier in the first row, as otherwise the maps would be tiny (like those first two graphs). However, it means the cadence chart is excessively big. Thoughts?

Not too sure tbh. Later on, clicking a graph will make that graph the 'big' map. For now, let's have either velocity or the power graph as the bigger graph since they're probably more useful than cadence.

**Review by harsilspatel — APPROVED** [review]
looks good to me, nice work @hallgchris ;)

**Review by khanguslee — APPROVED** [review]

**Comment by hallgchris** [comment]
> Not too sure tbh. Later on, clicking a graph will make that graph the 'big' map. For now, let's have either velocity or the power graph as the bigger graph since they're probably more useful than cadence.

Do I make another PR for this?

**Pull request merged**